README-en.md

```diff
@@ -0,0 +1,172 @@
+*This policy is a "living" document, and subject to refinement and expansion in the future. It was inspired by [Android Study Group Code of Conduct](https://github.com/AndroidStudyGroup/Code-Of-Conduct)*
+
+* [Code of conduct](#code-of-conduct)
+* [How to join?](#how-to-join)
+* [Good manners - Slack](#good-manners---slack)
+  * [Commands](#commands)
+  * [Questions](#questions)
+  * [Be kind](#be-kind)
+  * [Individual responsibility](#individual-responsibility)
+  * [Files and links](#files-and-links)
+  * [Channels](#channels)
+  * [Bots and plugins](#bots-and-plugins)
+  * [Get started with Slack](#get-started-with-slack)
+  * [Suggestions and improvements](#suggestions-and-improvements)
+
+## Code of conduct
+
+The **Android Dev BR** should be a safe place for everybody regardless of:
+
+  - Gender, gender identity or gender expression
+  - Sexual orientation
+  - Disability
+  - Physical appearance (including but not limited to body size)
+  - Ethnicity
+  - Age
+  - Religion
+
+As someone who is part of this Slack, you agree that:
+
+  - We are, collectively and individually, committed to safety and inclusivity.
+  - We have zero tolerance for abuse, harassment, or discrimination.
+  - We respect the boundaries and identity of people.
+  - We refrain from using language that can be considered oppressive (systemically or otherwise), eg. sexist, racist, homophobic, transphobic, ableist, classist, etc. - this includes (but is not limited to) various slurs.
+  - We avoid offensive topics as a form of humor.
+  - We do not tolerate piracy and violation of the terms of use of other software and services.
+
+We actively work towards:
+
+  - Be a safe community.
+  - Cultivate a network of support and encouragement for all.
+  - Encourage varied forms of expression in a responsible way.
+
+We condemn:
+
+  - Create profile by pretending to be someone else for entertainment purposes, fraud or encouraging code of conduct violations (i.e. impersonation).
+  - Persecution, "doxxing" (investigating the life of a person without authorization) or undue publication of private information.
+  - Threats of all kinds and harassment.
+  - Any behavior that compromises the safety of other members.
+
+**These things are NOT OK.**
+
+If you say something that is considered offensive and somebody call you out on your comment, it is important to:
+
+  - Listen without interrupting.
+  - Believe in what the person is saying and do not try to disqualify what the person has to say.
+  - Ask for tips or help to avoid future offenses.
+  - Apologize and ask forgiveness.
+
+### If you say something that is found offensive, and you are called out on it, let’s:
+
+* Listen without interruption.
+* Believe what the person is saying & do not attempt to disqualify what they have to say.
+* Ask for tips / help with avoiding making the offense in the future.
+* Apologize and ask forgiveness.
```

Comment:
Duplicou a parte desde `If you say something that is found `

Reply:
Não ficou duplicado essa parte não?